pkgs/applications/graphics/imlibsetroot/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, libX11, xorg, imlib2 }:
+
+stdenv.mkDerivation rec {
+  name = "imlibsetroot-${version}";
+  version = "1.2";
+  src = fetchurl {
+    url = "http://robotmonkeys.net/wp-content/uploads/2010/03/imlibsetroot-12.tar.gz";
+    sha256 = "8c1b3b7c861e4d865883ec13a96b8e4ab22464a87d4e6c67255b17a88e3cfd1c";
+  };
+
+  buildInputs = [ libX11 imlib2 xorg.libXinerama ];
+  buildCommand = ''
+    tar xf ${src}
+    gcc -g imlibsetroot-1.2/imlibsetroot.c                                         \
+      `${imlib2}/bin/imlib2-config --cflags` `${imlib2}/bin/imlib2-config --libs`  \
+      -I${libX11}/usr/include/X11/extensions -Iimlibsetroot-1.2                    \
+      -L${xorg.libXinerama}/lib -L${libX11}/lib                                    \
+      -lXinerama -lX11 -o imlibsetroot
+    mkdir -p $out/bin
+    install -m 755 imlibsetroot $out/bin
```

Comment:
I disagree : I don't use the default builder, so there is no phase. However, if you think it is better, I can split ```buildCommand``` between ```unpackPhase```, ```buildPhase``` and ```installPhase```.